src/blocks/file-blocks/excalidraw/index.tsx

```diff
@@ -1,52 +1,62 @@
 import { FileBlockProps } from "@githubnext/utils";
 import { useEffect, useState } from "react";
-import "./style.css"
+import "./style.css";
 
 if (typeof window !== "undefined") {
   // to load assets from self domain, instead of the CDN
   const urlObject = new URL(window.location.href);
-  const prototypeDomain = "https://next-devex-blocks.azurewebsites.net"
+  const prototypeDomain = "https://blocks.githubnext.com";
   const isPrototype = urlObject.origin === prototypeDomain;
   if (isPrototype) {
     // @ts-ignore
-    window.EXCALIDRAW_ASSET_PATH = "/excalidraw/"
+    window.EXCALIDRAW_ASSET_PATH = "/excalidraw/";
```

Comment:
what is this bit accomplishing?

Reply:
This is a relic, I didn't add this. From the docs – https://www.npmjs.com/package/@excalidraw/excalidraw
> After installation you will see a folder excalidraw-assets and excalidraw-assets-dev in dist directory which contains the assets needed for this app in prod and dev mode respectively.

Move the folder excalidraw-assets and excalidraw-assets-dev to the path where your assets are served.

By default it will try to load the files from https://unpkg.com/@excalidraw/excalidraw/dist/

If you want to load assets from a different path you can set a variable window.EXCALIDRAW_ASSET_PATH depending on environment (for example if you have different URL's for dev and prod) to the url from where you want to load the assets.